cmd/porter/inspect.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/deislabs/porter/pkg/porter"
+)
+
+func buildBundleInspectCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.ExplainOpts{}
+	cmd := cobra.Command{
+		Use:   "inspect",
+		Short: "Inspect a bundle",
```

Comment:
It would be nice if we provided hints on when to use explain and when to use inspect in the long helptext.